src/templates/base.html

```diff
@@ -31,6 +31,12 @@
     <div id="navigation">
       {% block navigation %}{% endblock %}
     </div>
+    <div class="language-switcher">
+      {% block current_lang %}{% endblock %}
```

Comment:
I'd love to see this generated dynamically if possible. We should know based on the current template what its current language is and we can pass in the `SUPPORTED_LANGUAGES` list to generate the alternate language options. This way as we add support for more languages, we don't need to update every page.